assets/src/scss/layout/_breadcrumbs.scss

```diff
@@ -25,6 +25,10 @@
       }
     }
     display: inline-block;
+
+    &:not(:only-child):not(:last-child) {
+      margin-inline-end: 7px;
```

Comment:
Shouldn't it be 8px instead of 7px? 🤔

Reply:
Ah yes, that would make more sense. I used the old value, but better use multiples of 8. Commit amended.